subprojects/smoke-test/src/smokeTest/groovy/org/gradle/play/integtest/fixtures/external/AbstractMultiVersionPlayExternalContinuousBuildIntegrationTest.groovy

```diff
@@ -34,6 +35,9 @@ abstract class AbstractMultiVersionPlayExternalContinuousBuildIntegrationTest ex
     def setup() {
         buildFile << playPlatformConfiguration(version.toString())
         executer.expectDeprecationWarning("The ProjectLayout.configurableFiles() method has been deprecated. This is scheduled to be removed in Gradle 7.0. Please use the ObjectFactory.fileCollection() method instead.")
+
+        // Play Framework supports up to Java 11.
+        Assume.assumeTrue(JavaVersion.current() <= JavaVersion.VERSION_11)
```

Comment:
Why was this not needed before?

Reply:
Things might have _worked_ though its not supported. 
The docs say the newest version of play supports 8 to 11: https://www.playframework.com/documentation/2.8.x/Requirements

We're only testing a subset of play, I'm sure there are other places which are unsupported that we don't have coverage for.